### ISSUE TITLE
Buggs with Yuggs

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#75062]
### Describe the bug

Yugg have a "gene_sting" spell now that they cast on the player instead of their hardcoded dart fake-gun attack.
The implementation is problematic: it does "biological" damage, against which there is no resistance (no armor or effect provides resistance to it). It has a range of 24, leading to a guaranteed mutation. I believe you also used to be able to dodge the dart, you cannot dodge a spell.
Originally, it was a "dart", that could be blocked by armor, but even if the attack is made to do piercing damage instead of biological damage, it still won't fix the issue because the spell it casts on impact will happen even if the player wholly absorbs the damage the mutation still happens.

### Attach save file

n/a

### Steps to reproduce

1. Read the json for yuggs.

### Expected behavior

Presumably yuggs are supposed to be actually firing a dart at the player that can bounce off their armor and not mutate them, rather than casting a Malmutate spell like Orbs of Fire or Neqoxecs from DCSS. So they should probably just be using the hardcoded gene_sting monster attack because that has additional code to handle stuff like this.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.4651 (22H2)
- Game Version: b87d515 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Slowdown Fungal Growth [no_fungal_growth],
    Bionic Slots [cbm_slots],
    Portal Storms Ignore NPCs [personal_portal_storms]
]

**Comment by Maleclypse** [comment]
So anything you want to do to fix it other than going back to the hardcoded sting will be fine. The hardcoded sting breaks in other ways.

**Comment by Maleclypse** [comment]
Commenting here that yuggs shouldn't make footstep noises and that I'll need to move them to a burrowing sound of some sort while I'm working on this issue.

**Comment by Maleclypse** [comment]
Fixed by #75079
